packages/pluggableWidgets/datagrid-web/src/components/Header.tsx

```diff
@@ -30,6 +31,9 @@ export function Header<D extends object>(props: HeaderProps<D>): ReactElement {
     const canSort = props.sortable && props.column.canSort;
     const canDrag = props.draggable && (props.column.canDrag ?? false);
     const draggableProps = useDraggable(canDrag, props.visibleColumns, props.setColumnOrder, props.setDragOver);
+    const totalWeight = useMemo(() => props.visibleColumns.reduce((p, c) => p + (c.weight ?? 1), 0), [
```

Comment:
Hmm, so we give weight 1 to both auto-grow and auto-fit columns? Is that correct?

Reply:
Yes, because we consider just the number of active columns or if a column has higher weight we consider the weight as the column numbers.